.github/workflows/linux.yml

```diff
@@ -166,7 +170,7 @@ jobs:
 
       - name: Build (${{ inputs.profile }})
         run: |
-          ./mach build --use-crown --locked --${{ inputs.profile }}
+          ./mach build --use-crown --locked --${{ inputs.profile }} ${{ inputs.build-args }}
```

Comment:
Where are the build args going to come from? Are they just going to be statically defined in the workflows and this is just setting up the interesting for that?  Im concerned about the risk of [template injection](https://docs.zizmor.sh/audits/#template-injection) here

Reply:
like all inputs, they will come from workflow call which populates args based on try_parser.py output. No injection should be possible (at least from non servo org members). But if it is possible it's already possible via other inputs (here it would be possible to abuse profile to get functionality of build-args).